src/GovUk.Education.ExploreEducationStatistics.Notifier.Model/GovUk.Education.ExploreEducationStatistics.Notifier.Model.csproj

```diff
@@ -8,4 +8,8 @@
     <ItemGroup>
         <PackageReference Include="IsExternalInit" Version="1.0.3" PrivateAssets="all" />
     </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Common\GovUk.Education.ExploreEducationStatistics.Common.csproj" />
+    </ItemGroup>
```

Comment:
Would be interesting to know if this has pumped up the size of the deployment package significantly or not.

This does have a lot of dependencies that won't be relevant to the Notifier, for the sake of sharing the `IdTitleViewModel`.

We will probably benefit from using more shared code in future too so I don't think it's an issue but just asking more out of curiosity.